Dockerfile

```diff
@@ -114,7 +115,7 @@ RUN make $MAKEFLAGS && \
 FROM ubuntu:bionic AS cpiface
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-        libgoogle-glog0v5 \
+        libgoogle-glog0v5 libjsoncpp-dev \
```

Comment:
if linnking to static version is possible lets do that.
else change this to libjsoncpp1

Reply:
Done.